src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -74,6 +77,10 @@ public Subsystems() {
 				// YUYV only:
 				// 2304x1296, 2304x1536
 				driverVisionCamera.setResolution(160, 120);
```

Comment:
We might consider starting with a higher resolution so we have it fully vetted for bandwidth problems as soon as we can.  I believe the off season FMS is setup to limit bandwidth.